.idea/codeStyleSettings.xml

```diff
@@ -3,10 +3,28 @@
   <component name="ProjectCodeStyleSettingsManager">
     <option name="PER_PROJECT_SETTINGS">
       <value>
-        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+        <option name="OTHER_INDENT_OPTIONS">
```

Comment:
Thanks, I do need to add to it: `<option name="BLANK_LINES_BEFORE_PACKAGE" value="1" />`